cmd/gen-manpages/main.go

```diff
@@ -41,20 +41,23 @@ func run() error {
 		"containerd": command.App(),
 		"ctr":        app.New(),
 	}
-	dir := flag.Arg(0)
-	for name, app := range apps {
-		// clear out the usage as we use banners that do not display in man pages
-		app.Usage = ""
-		data, err := app.ToMan()
-		if err != nil {
-			return err
-		}
-		if _, err := os.Stat(dir); os.IsNotExist(err) {
-			os.Mkdir(dir, os.ModePerm)
-		}
-		if err := ioutil.WriteFile(filepath.Join(dir, fmt.Sprintf("%s.1", name)), []byte(data), 0644); err != nil {
-			return err
-		}
+	name := flag.Arg(0)
+	dir := flag.Arg(1)
+	app, ok := apps[name]
+	if !ok {
+		return fmt.Errorf("Invalid application '%s'", name)
```

Comment:
Guess this should've been lowercase

Reply:
I though the linters were complaining about this, maybe just a warning. Should fix upstream though if we have to, fine for now

Reply:
Yes; this is just a cherry-pick, but was also surprised the linter didn't make it fail